modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/dto/v1_0/util/ObjectLayoutUtil.java

```diff
@@ -68,12 +72,17 @@ public static ObjectLayout toObjectLayout(
 
 	public static ObjectLayoutTab toObjectLayoutTab(
 		ObjectFieldLocalService objectFieldLocalService,
-		com.liferay.object.model.ObjectLayoutTab objectLayoutTab) {
+		com.liferay.object.model.ObjectLayoutTab objectLayoutTab,
+		ObjectRelationshipLocalService objectRelationshipLocalService) {
 
 		if (objectLayoutTab == null) {
 			return null;
 		}
 
+		ObjectRelationship objectRelationship =
+			objectRelationshipLocalService.fetchObjectRelationship(
+				objectLayoutTab.getObjectRelationshipId());
+
```

Comment:
@marcelabc is this code necessary? You already did this [here](https://github.com/liferay-objects/liferay-portal/blob/ad02ab4dce86add0cc281e35c9ebf15b53562b3a/modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/dto/v1_0/util/ObjectLayoutUtil.java#L102).